Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,242 @@
+        {
+            if (numberOfPeople > 1)
+            {
+                if (amount < 0)
+                {
+                    return people + " have the least money. -¤" + Math.Abs(amount) + ".";
+                }
+                else
+                {
+                    return people + " have the least money. ¤" + amount + ".";
+                }
+
+            }
+            else
+            {
+                if (amount < 0)
+                {
+                    return people + " has the least money. -¤" + Math.Abs(amount) + ".";
+                }
+                else
+                {
+                    return people + " has the least money. ¤" + amount + ".";
+                }
+
+            }
+        }
+        /// <summary>
+        /// Gives back a string that reads: people lost the most money. ¤ amount.
+        /// </summary>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringBiggestLossPeople(string people, decimal amount)
+        {
+            if (people == "")
+            {
+                return "N/A.";
+            }
+            if (amount < 0)
+            {
+                return people + " lost the most money. -¤" + Math.Abs(amount) + ".";
+            }
+            else
+            {
+                return people + " lost the most money. ¤" + amount + ".";
+            }
+
+        }
+        /// <summary>
+        /// Gives back a string that reads: people had the most money ever. ¤ amount.
+        /// </summary>
+        /// <param name="numberOfPeople"></param>
+        /// <param name="people">string of person or persons</param>
+        /// <param name="amount">the amount of money</param>
+        /// <returns></returns>
+        public static string MakefullstringHighestBalanceEver(int numberOfPeople, string people, decimal amount)
+        {
+            if (people == "")
+            {
+                return "N/A.";
+            }
+            if (numberOfPeople > 1)
+            {
+                return people + " had the most money ever. ¤" + amount + ".";
+            }
+            else
+            {
+                return people + " had the most money ever. ¤" + amount + ".";
+            }
+
+        }
+        public static string MakePadding(string message, int padding)
+        {
+            const string corner = "+";
+            const string horizontal = "-";
+            const string vertical = "|";
+
+            string[] splitmessage = message.Split(Environment.NewLine);
+            int height = splitmessage.Length + (padding * 2) + 1;
+            int width = 0;
+
+            foreach (string line in splitmessage)
+            {
+                if(line.Length > width)
+                {
+                    width = line.Length;
+                }
+            }
+            width += (padding * 2) + 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'width'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)